packages/ui/product-info/style.ts

```diff
@@ -0,0 +1,222 @@
+import { Box, Button, Container, Text } from '@mantine/core';
+import styled from 'styled-components';
+
+const maxMobileWidth = '976px';
+const sm = '600px';
+const md = '900px';
+const lg = '1200px';
```

Comment:
upon checking this sizes can be move to constant to be used to all styling

Reply:
done. please check. thanks!